RELEASE.md

```diff
@@ -0,0 +1,21 @@
+# Process for creating a Kube-Router release
+
+## Preparing for the release
+* Ensure that the Golang release used is still supported.
+* Ensure that the Alpine version used for containers is still supported
+* Ensure that Golang dependencies are updated.
+* Ensure that the GoBGP version is updated.
+* Ensure that the Kubernetes object definitions do not contain deprecated object types.
+
+## New major/minor release
+* Create a new release on Github from the default branch (currently: master) with the release tag v$MAJOR.$MINOR.0
+* Create a branch named v$MAJOR.$MINOR
```

Comment:
Good release practices would dictate that we cut the branch first and then release from the branch. Then the process for patch release and major/minor release are always cut from the branch, right?

Reply:
Also, there needs to be a step to go into the GitHub releases and approve the draft release that goreleaser creates right?

Reply:
Yes, added those steps. Thanks for catching them!